docs/jwst/user_documentation/running_pipeline_python.rst

```diff
@@ -529,3 +529,166 @@ individual step parameter must be set when using this method, or else the coded
 defaults will be used, which may be inappropriate for the dataset being processed.
 
 See :ref:`call_examples` for more information.
+
+
+.. _multiprocessing:
+
+Multiprocessing
+===============
+
+Multiprocessing is supported to speed up certain computationally-intensive steps
+in the pipeline, including :ref:`jump_step <jump_step>`,
+:ref:`ramp_fitting_step <ramp_fitting_step>`, and
+:ref:`wfss_contam_step <wfss_contam_step>`. The examples below show how
+multiprocessing can be enabled for these steps, as well as how to set up
+multiprocessing to simultaneously run the entire pipeline on multiple observations.
+
+Python's multiprocessing module explicitly imports and executes a script's
+`__main__` with each and every worker. If `__main__` is not present the behavior is
```

Comment:
Is this still valid or did you find out now that including `main` is not necessary?

Reply:
This comment is still valid. The __main__ was not necessary in the Class files but it is necessary in the code that runs the multiprocessing call.